### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
### Description

Fixes misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/radis/actions/runs/5865276334#summary-15901868579

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/radis/actions/runs/5865276395#summary-15901868888

requested by @erwanp in https://github.com/hitranonline/hapi/pull/42#issuecomment-1674824161

### CODE REVIEW

File: radis/tools/slit.py [hunk 1]
```diff
@@ -1630,7 +1630,7 @@ def triangular_slit(
     return w, I
 
 
-# trapezoidal instrumental broadening function of base base nm and top top nm
+# trapezoidal instrumental broadening function of base (nm) and top (nm)
```
Comment (jsoref) [inline radis/tools/slit.py:1633]: In general I like my duplicate detection... This case is interesting. I think it's trying to describe the identify of things, their variables, and their types. I don't think writing it that way is helpful and so I'm proposing an alternative. I'm not attached to it. My tool can be tuned to ignore this content...

File: docs/references/_constants.rst [hunk 2]
```diff
@@ -6,7 +6,7 @@ Spectroscopic constants
 CO2
 ---
 
-Version 0.9 uses uses the CO2 spectrosopic coefficients of [Klarenaar2017]_ ,
+Version 0.9 uses the CO2 spectrosopic coefficients of [Klarenaar2017]_ ,
```
[the remark this answers is not in the document]
Comment (erwanp) [reply docs/references/_constants.rst:9]: Nice

File: examples/plot_linestrengths.py [hunk 3]
```diff
@@ -61,7 +61,7 @@ def _calc_Q(molecule, iso, T_ref, T_gas):
         Qref, Qgas = _calc_Q(molecule, iso, Tref, Tgas)
         Qref_Qgas_ratio[iso] = Qref / Qgas
 
-    # Scaling linestrength with the equations from Rotham's paper
+    # Scaling linestrength with the equations from Rothman's paper
```
[the remark this answers is not in the document]
Comment (erwanp) [reply examples/plot_linestrengths.py:64]: Yes, it's his right name

File: radis/cython/radis_cython_extensions.pyx [hunk 4]
```diff
@@ -120,7 +120,7 @@ def calc_lorentzian_envelope_params(
     cdef size_t top_size = 0
     cdef size_t bottom_size = 0
 
-    fname = "Lorenzian_minmax_" + str(len(gamma)) + ".dat"
+    fname = "Lorentzian_minmax_" + str(len(gamma)) + ".dat"
```
[the remark this answers is not in the document]
Comment (erwanp) [reply radis/cython/radis_cython_extensions.pyx:123]: Yes, good correction

File: radis/db/classes.py [hunk 5]
```diff
@@ -783,12 +783,12 @@ def _assign_E(self, Erovib, Ehaj):
...
 
         if Erovib is not None:
-            # overwrite energy calculation
+            # override energy calculation
             self.Erovib = Erovib
             self.Ehaj = Ehaj
             if self.verbose >= 2:
                 print(
-                    "{0}: overwritting Energy calculation with {1}".format(
+                    "{0}: overriding Energy calculation with {1}".format(
```
[the remark this answers is not in the document]
Comment (erwanp) [reply radis/db/classes.py:791]: Fine for me!

File: radis/db/conventions.py [hunk 6]
```diff
@@ -27,7 +27,7 @@
 ## Invert the dictionary.
 herzberg2dunham = {v: (sign, k) for k, (sign, v) in dunham2herzberg.items()}
 """ dict: {Yij: (sign, coeff)
-conversion of Herberg convention to Dunham spectroscopic coefficients
+conversion of Herzberg convention to Dunham spectroscopic coefficients
```
[the remark this answers is not in the document]
Comment (erwanp) [reply radis/db/conventions.py:30]: Good correction, Nobel Prize 1971!

File: radis/lbl/base.py [hunk 7]
```diff
@@ -2242,7 +2242,7 @@ def calc_linestrength_eq(self, Tgas):
                 df1.int
                 * self.Qref_Qgas_ratio(df1, Tgas, Tref)
                 *
-                # ratio of Boltzman populations
+                # ratio of Boltzmann populations
```
[the remark this answers is not in the document]
Comment (erwanp) [reply radis/lbl/base.py:2245]: Good fix (although Boltzmann never had a Nobel Prize!)

File: radis/lbl/broadening.py [hunk 8]
```diff
@@ -964,7 +964,7 @@ def _calc_broadening_HWHM(self):
         self.profiler.stop("calc_hwhm", "Calculate broadening HWHM")
 
     def _calc_min_width(self, df):
-        """Calculates the minimum FWHW of the lines
+        """Calculates the minimum FWHM of the lines
```
[the remark this answers is not in the document]
Comment (erwanp) [reply radis/lbl/broadening.py:967]: Yes, good catch,Full Width at Half Maximum

**Review by erwanp — APPROVED** [review]
Thanks!

**Pull request merged**